gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/resources/webroot/views/forgot_password.html

```diff
@@ -8,10 +8,12 @@
     <title>Forgot password</title>
 
     <!-- CSS -->
-    <link rel="stylesheet" th:href="@{assets/material/material.blue_grey-blue.min.css}">
-    <link rel="stylesheet" th:href="@{assets/material/material.icons.css}">
-    <link rel="stylesheet" th:href="@{assets/font-awesome/css/font-awesome.min.css}">
-    <link rel="stylesheet" th:href="@{assets/css/forgot_password.css}">
+    <link rel="stylesheet" th:href="@{assets/css/login_flow.css}">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700;900&display=swap" rel="stylesheet">
```

Comment:
I'm not a huge fan of calling external assets (for enterprise proxy issue mainly).

Is it possible to use the font-face CSS feature ? something like : 

```
in the main.css file : 

 /* get the required local files */
    @font-face {
      font-family: 'Roboto';
      src: url('roboto.eot'); /* IE9 Compat Modes */
      src: url('roboto.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
      url('roboto.woff') format('woff'), /* Modern Browsers */
      url('roboto.ttf')  format('truetype'), /* Safari, Android, iOS */
      url('roboto.svg#svgFontName') format('svg'); /* Legacy iOS */
    }
    
    /* use the font */
    body {
      font-family: 'Roboto', sans-serif;
      font-size: 48px;
    }
```
More information here : https://stackoverflow.com/questions/18178867/how-can-i-use-googles-roboto-font-on-a-website

```

Reply:
done